index.js

```diff
@@ -36,45 +43,51 @@ const findIn = (root, ...tags) => {
 	return el
 }
 
-const parsePosList = (_, transformCoords, stride = 2) => {
+const createChildContext = (_, opts, ctx) => {
+	const srsDimensionAttribute = _.attributes && _.attributes.srsDimension
+	if (srsDimensionAttribute) {
+		const srsDimension = parseInt(srsDimensionAttribute)
+		if (Number.isNaN(srsDimension) || srsDimension <= 0) throw new Error(`invalid srsDimension attribute value "${srsDimensionAttribute}", expected a positive integer`)
+
+		const childCtx = Object.create(ctx)
+		childCtx.srsDimension = srsDimension
+		return childCtx;
+	} else {
```

Comment:
Drop the else. more info: http://blog.timoxley.com/post/47041269194/avoid-else-return-early

Reply:
How would you write it?

Reply:
```
const createChildContext = (_, opts, ctx) => {
	const srsDimensionAttribute = _.attributes && _.attributes.srsDimension
	if (!srsDimensionAttribute) return ctx; 

	const srsDimension = parseInt(srsDimensionAttribute)
	if (Number.isNaN(srsDimension) || srsDimension <= 0) {
		throw new Error(`invalid srsDimension attribute value "${srsDimensionAttribute}", expected a positive integer`)
	}

	const childCtx = Object.create(ctx)
	childCtx.srsDimension = srsDimension
	return childCtx;
}
```

Reply:
I usually move the narrower/edge case into an `if` clause. Therefore, in this case, I'd do this:

```js
const createChildContext = (_, opts, ctx) => {
	const srsDimensionAttribute = _.attributes && _.attributes.srsDimension
	if (srsDimensionAttribute) {
		const srsDimension = parseInt(srsDimensionAttribute)
		if (Number.isNaN(srsDimension) || srsDimension <= 0) throw new Error(`invalid srsDimension attribute value "${srsDimensionAttribute}", expected a positive integer`)

		const childCtx = Object.create(ctx)
		childCtx.srsDimension = srsDimension
		return childCtx;
	}

	return ctx;
}
```